apis/config/v1alpha1/cluster_colocation_profile_types.go

```diff
@@ -52,6 +50,26 @@ type ClusterColocationProfileSpec struct {
 	// +optional
 	SchedulerName string `json:"schedulerName,omitempty"`
 
+	// QoSClass describes the type of koordinator QoS that the Pod is running.
+	// Can be LSE/LSR/LS/BE/SYSTEM.
+	// +kubebuilder:validation:Enum=LSE;LSR;LS;BE;SYSTEM
+	// +optional
+	QoSClass string `json:"qosClass,omitempty"`
+
+	// If specified, indicates the pod's priority.
+	// Optional can be specified koordinator-prod/koordinator-batch/koordinator-mid/koordinator-free
```

Comment:
change the order koordinator-mid/koordinator-batch

Reply:
if possible use `+kubebuilder:validation:Enum`